src/mmfatl.c

```diff
@@ -71,29 +153,110 @@ static void initBuffer(void) {
 
 bool test_initBuffer(void) {
   // emulate memory corruption
-  memset(buffer, 'x', sizeof(buffer));
+  memset(&buffer, 'x', sizeof(buffer));
 
   initBuffer();
 
+  ASSERT(buffer.begin == buffer.text)
```

Comment:
```suggestion
  ASSERT(buffer.begin == buffer.text);
```
Use semicolon after statement-like macros